canonical-websites/www.ubuntu.com
# New /kubernetes page

## Done
Created a new /kubernetes page

[Demo](http://www.ubuntu.com-pr-2179.run.demo.haus/kubernetes)

[Copydoc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#heading=h.t6q2xj6x8p2i)

## QA
- `./run`
- In a browser, visit: [http://localhost:8001/kubernetes](http://localhost:8001/kubernetes)
- Check against the [copydoc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#)
- Check the redirects like: [http://localhost:8001/containers/kubernetes](http://localhost:8001/containers/kubernetes)

Fixes #2163
